src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```diff
@@ -180,5 +180,20 @@
       "RedisStreamTrigger",
       "RedisListTrigger"
     ]
+  },
+  {
+    "id": "Microsoft.Azure.WebJobs.Extensions.Dapr",
+    "majorVersion": "0",
+    "name": "Dapr",
+    "bindings": [
+      "daprBinding",

```

Comment:
Are these all binding types?

Reply:
Yes. Here are [binding types](https://github.com/Azure/azure-functions-dapr-extension/tree/master/src/Microsoft.Azure.Functions.Worker.Extensions.Dapr/Bindings), and [trigger types](https://github.com/Azure/azure-functions-dapr-extension/tree/master/src/Microsoft.Azure.Functions.Worker.Extensions.Dapr/Triggers).